e2e/deployers/crud.go

```diff
@@ -112,7 +112,7 @@ func CreatePlacement(ctx types.Context, name, namespace string) error {
 			return err
 		}
 
-		log.Debugf("Placement \"%s/%s\" already Exists", namespace, name)
+		log.Debugf("Placement \"%s/%s\" already exists", namespace, name)
```

Comment:
It will be useful to mention these changes in the commit message (unify captialization in some logs).

If there are many changes it can be useful to have them in a separate commit.